qemu/tests/cfg/live_snapshot.cfg

```diff
@@ -73,3 +73,11 @@
                 - stop_cont:
                     before_start = "stop"
                     after_finished = "resume reboot verify_alive"
+        - negative_test:
+            type = live_snapshot_negative_test
+            variants:
+            - @default:
+                match_str = "Could not create file: No such file or directory"
+            - existing_mode:
```

Comment:
Thanks.
It's a mistake after rebase.

This problem has been fixed, could you help review it again?

Thanks.